### ISSUE TITLE
Problem with multible Host rules

### ISSUE THREAD

**Issue by bikaro** [hardillb/traefik-avahi-helper#3]
Thx for fixing the other bug so quickly, found an other one for you in my setup:

When there are multiple host rules in on label, it tries to add all the rules as one string as a domain which obvious does not work.

Here the error log:
```
> traefik-avahi-helper@1.0.1 start /usr/src/app
> node index.js

[
  'syncthing.myname.local',
  'traefik.myname.local`) && (PathPrefix(`/api`) || PathPrefix(`/dashboard',
  'git.myname.local'
]
starting cname.py
Traceback (most recent call last):
  File "cname.py", line 93, in <module>
    main()
  File "cname.py", line 81, in main
    status = publisher.publish_cname(cname, True)
  File "/usr/local/lib/python3.8/site-packages/mpublisher/mpublisher.py", line 117, in publish_cname
    group.AddRecord(avahi.IF_UNSPEC, avahi.PROTO_UNSPEC, dbus.UInt32(0), cname.encode("ascii"),
  File "/usr/local/lib/python3.8/site-packages/dbus/proxies.py", line 72, in __call__
    return self._proxy_method(*args, **keywords)
  File "/usr/local/lib/python3.8/site-packages/dbus/proxies.py", line 141, in __call__
    return self._connection.call_blocking(self._named_service,
  File "/usr/local/lib/python3.8/site-packages/dbus/connection.py", line 652, in call_blocking
    reply_message = self.send_message_with_reply_and_block(
dbus.exceptions.DBusException: org.freedesktop.Avahi.NotSupportedError: Not supported
```

The used label:
```
- "traefik.http.routers.dashboard.rule=Host(`traefik.myname.local`) && (PathPrefix(`/api`) || PathPrefix(`/dashboard`))"
```

You could also imagine that people would use two domains in there rules and like both to be added. Or one .local and a public domain and then only the .local domain should be added. Maybe search for .local in the string given by `r2` in index.js to identify the correct rule/domain(s).

**Comment by hardillb** [comment]
Feel free to submit a pull request with a regex that will match all the possible rule combinations.

**Comment by bikaro** [comment]
Fixed with commit 7ba2a75
